src/LocalizedStrings.js

```diff
@@ -288,4 +292,32 @@ export default class LocalizedStrings {
   getContent() {
     return this._props;
   }
+
+  _runLanguageListeners(language) {
+    this._languageListeners.forEach((listener) => {
+      try {
+        listener(language);
+      } catch (error) {
+        /* no op */
+      }
+    });
+  }
+
+  addLanguageChangeListener(listener) {
+    this._languageListeners.push(listener);
+  }
+
+  removeLanguageChangeListener(listener) {
+    const index = this._languageListeners.findIndex((a) => a === listener);
```

Comment:
findIndex not supported on IE11, do we care? :)

Reply:
I care :D, updated to use filter instead :)